Source/Fuse.Controls.CameraView/Android/Camera.uno

```diff
@@ -155,6 +155,32 @@ namespace Fuse.Controls.Android
 			return new AndroidPhotoOptionPromise(this).Visit(options);
 		}
 
+		class AndroidCameraFocusPointPromise : Promise<CameraFocusPoint> {
```

Comment:
We don't use egyptian braces in Uno, as per [the coding style](https://github.com/fuse-open/fuselibs/blob/master/Documentation/CodingStyle.md). Please fix this elsewhere in this PR also.

Reply:
Aight, I think I'm getting the styles now...